warehouse/oidc/views.py

```diff
@@ -238,6 +238,16 @@ def mint_token(
         publisher = oidc_service.find_publisher(claims, pending=False)
         # NOTE: assert to persuade mypy of the correct type here.
         assert isinstance(publisher, OIDCPublisher)
+    except ReusedTokenError:
+        return _invalid(
+            errors=[
+                {
+                    "code": "invalid-reuse-token",
+                    "description": "valid token, but already used",
```

Comment:
Random thought: we might want to make this error message match the general "token invalid" message, to avoid giving an attacker an oracle for whether the token has been reused. I'm not sure that matters in practice though (since the reused token is useless), and may just end up confusing users.

CC @di for thoughts 🙂

Reply:
I don't think it matters, an invalid token and a reused token are equally useless. I'm more concerned about telling users who are legitimately trying to reuse a token that token reuse is not permitted.